Core/CategoriesManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Core
+{
+    public class CategoriesManager
+    {
+		public static Dictionary<string, Func<Transaction, bool>> Load()
+		{
+			if (!File.Exists("categories.json"))
+			{
+				return new Dictionary<string, Func<Transaction, bool>>();
+			}
+
+			var categories = JsonConvert.DeserializeObject<List<CategoryDTO>>(File.ReadAllText("categories.json"));
+
+			var categoryFilters = new Dictionary<string, Func<Transaction, bool>>();
+
+			foreach (var c in categories)
+			{
+				categoryFilters.Add(c.Name, t => Regex.IsMatch(t.ApplicationCode, '@' + c.ApplicationCode)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Reduce the number of conditional operators (5) used in the expression (maximum allowed 3).](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4966361)